CMake error: Unknown CHECK_C_COMPILER_FLAG

```
$ cmake --version
cmake version 3.28.2

$ cmake .. -DCMAKE_BUILD_TYPE=Release -DUSE_SYSTEM_ZLIB=true -DUSE_SYSTEM_OPENAL=true -DUSE_SYSTEM_CURL=true -DUSE_SYSTEM_SDL2=true
CMake Deprecation Warning at extern/glslang/CMakeLists.txt:39 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Warning (dev) at extern/glslang/CMakeLists.txt:233 (find_package):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
  the cmake_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  extern/glslang/CMakeLists.txt:239 (find_host_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Using glslangValidator built from source
CMake Error at src/CMakeLists.txt:399 (CHECK_C_COMPILER_FLAG):
  Unknown CMake command "CHECK_C_COMPILER_FLAG".
Call Stack (most recent call first):
  src/CMakeLists.txt:408 (CCompilerFlagString)
  src/CMakeLists.txt:415 (CCompilerFlagStringAppend)
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -394,6 +394,7 @@ if(NOT WIN32)
     add_compile_definitions(_GNU_SOURCE)
 endif()
 
+include(CheckCCompilerFlag)
 # Check whether compiler supports 'flagstr', and if so, set ${flagvar} to that value
 function(CCompilerFlagString flagvar flagstr)
     CHECK_C_COMPILER_FLAG("${flagstr}" ${flagvar}_FOUND)

